sdk/nodejs/output.ts

```diff
@@ -19,120 +19,150 @@ import * as utils from "./utils";
 /* eslint-disable no-shadow, @typescript-eslint/no-shadow */
 
 /**
- * Output helps encode the relationship between Resources in a Pulumi application. Specifically an
- * Output holds onto a piece of Data and the Resource it was generated from. An Output value can
- * then be provided when constructing new Resources, allowing that new Resource to know both the
- * value as well as the Resource the value came from.  This allows for a precise 'Resource
- * dependency graph' to be created, which properly tracks the relationship between resources.
+ * {@link Output} helps encode the relationship between {@link Resource}s in a
+ * Pulumi application. Specifically, an {@link Output} holds onto a piece of
+ * data and the resource it was generated from. An output value can then be
+ * provided when constructing new resources, allowing that new resource to know
+ * both the value as well as the resource the value came from.  This allows for
+ * a precise resource dependency graph to be created, which properly tracks the
+ * relationship between resources.
  */
 class OutputImpl<T> implements OutputInstance<T> {
     /**
      * A private field to help with RTTI that works in SxS scenarios.
      *
      * This is internal instead of being truly private, to support mixins and our serialization model.
+     *
      * @internal
      */
     // eslint-disable-next-line @typescript-eslint/naming-convention,no-underscore-dangle,id-blacklist,id-match
     public readonly __pulumiOutput: boolean = true;
 
     /**
-     * Whether or not this 'Output' wraps a secret value. Values which are marked as secret are stored in an
-     * encrypted format when they are persisted as part of a state file. When`true` this "taints" any
-     * additional resources created from it via an [all] or [apply], such that they are also treated as
-     * secrets.
+     * Whether or not this output wraps a secret value. Values which are
```

Comment:
At what character count are these re-wrapped now?  Should we make this a standard for comments and write it down somewhere?  So far I've mostly just had the long line linter rewrap stuff, but if we agree on a number I can set my editor up to do that.

It's fine now, but I'd have preferred rewrapping only to be separate from other changes, since it makes it a little harder to see where there's only re-wrapping and where something actually changed.